src/Componenets/Signin/Signin.js

```diff
@@ -1,15 +1,154 @@
 import React, { Component } from "react";
 import "bootstrap/dist/css/bootstrap.css";
-import './Signin.css';
-
+import "./Signin.css";
 
 class Signin extends Component {
-	render(){
+	render() {
 		return (
 			// Signin goes here
 			<main>
-                {/* Write your Signin here */}
-                test signin
+				<div className="maincontainer" style={{ margin: "80px" }}>
+					<div className="container">
+						<div className="row justify-content-center">
+							<div className="col-md-6 col-md-offset-4">
+								<div className="card">
+									<div className="card-header">
+										<div className="container">
+											<div className="row">
+												<div
+													className="col-xs-6 col-md-6 col-lg-6 signup text-center"
+													id="signup-link"
+												>
+													<a
+														href="#signup"
+														className="active"
+														id="signup"
+													>
+														<strong>Sign Up</strong>
+													</a>
+												</div>
+												<div
+													className="col-xs-6 col-md-6 col-lg-6 signin text-center"
+													id="signin-link"
+												>
+													<a
+														href="#signin"
+														id="signin"
+													>
+														<strong>Sign In</strong>
+													</a>
+												</div>
+											</div>
+										</div>
+									</div>
+									<div className="card-body">
+										<div className="container">
+											<div className="row justify-content-center">
+												<div className="form-container col-md-10">
+													<form
+														onSubmit=""
+														action=""
```

Comment:
@Ajayjam97 Can you comment this `onSubmit` out? Since we arent submitting it yet, and it expects a function. It's giving error in the console.
Everything looks great 👍

Reply:
yes! sure.
Doing it.